client/app/pages/dashboards/dashboard.js

```diff
@@ -48,34 +48,37 @@ function DashboardCtrl(
 ) {
   this.saveInProgress = false;
 
-  const saveDashboardLayout = (widgets) => {
+  const saveDashboardLayout = () => {
     if (!this.dashboard.canEdit()) {
       return;
     }
 
+    // calc diff, bail if none
+    const changedWidgets = getWidgetsWithChangedPositions(this.dashboard.widgets);
+    if (!changedWidgets.length) {
```

Comment:
Lodash `isEmpty` is safer, no?

Reply:
In this case I know for sure it's gonna be an array.

<img src="https://user-images.githubusercontent.com/486954/56243809-27ca8e80-60a4-11e9-92d2-5d3a95255b9f.jpeg" width=500 />